greenback-kit-core/src/main/java/com/greenback/kit/client/impl/AbstractGreenbackClient.java

```diff
@@ -751,33 +751,33 @@ abstract protected AutoExport deleteAutoExportByUrl(
     //
 
     @Override
-    public AutoExportRun createAutoExportRun(String autoExportId, AutoExportRun autoExportRun) throws IOException {
-        Objects.requireNonNull(autoExportRun, "autoExportRun was null");
+    public ExportRun createExportRun(String autoExportId, ExportRun exportRun) throws IOException {
```

Comment:
This particular call actually is a "createAutoExportRun" since it specifically calls the auto exports runs endpoint.